### DISCUSSION TITLE
Exposing containers ports through IPv6 tunnel doesn't work.

### DISCUSSION THREAD

**Discussion by GamePlayer-8** [containers/podman#17606] [Q&A] [answered]
Heyo,

I'm having a bit of issues exposing container ports onto IPv6 network.
I'm using [HE](https://tunnelbroker.net) tunnelbroker. 

Ports what are running on the localhost are reachable through tunnel, but ports exposed from containers didn't.

I'm using CNI.

**Comment by GamePlayer-8** [comment]
Although containers ports on IPv6 IP are being shown as `closed` rather than `filtered` or empty.

**Comment by GamePlayer-8** [comment, marked as answer]
Problem has been solved by adding another argument `-p [::]:25:25` rather than only `-p 25:25`.

**Comment by GamePlayer-8** [comment]
Also there might be required to add in `/etc/containers/containers.conf`:
```
enable_ipv6=true
outbound_add6="::1"
```

**Reply by rhatdan** [reply]
Is this something we should write up in a blog?  @Luap99 WDYT?
